ui/src/main.css

```diff
@@ -82,11 +126,9 @@ a.button {
 }
 
 .well {
-  --light: #f5f5f5;
```

Comment:
I think this would break the .well-error and .well-warn classes below?

It's been a while, but I think you can exercise `warn` with a MIME error (should be part of swaks test mails), and `error` by killing the backend so GETs timeout.

Reply:
Sorry I just missed the notifications. I will check your suggestions over the weekend.

Reply:
You where right, now all should be fixed. The only thing that im not happy with it's the warning color, I couldn't find a better one.

![Screenshot from 2021-07-20 00-20-39](https://user-images.githubusercontent.com/429848/126257172-50fad847-525d-47b5-8529-507d41488002.png)

![Screenshot from 2021-07-20 00-21-22](https://user-images.githubusercontent.com/429848/126257182-f8255dad-c3f1-4944-8646-f51033dc7ff4.png)

Reply:
Hehe, yeah it's not an inspiring color.  But I don't have any better ideas, and it stands out well enough.